pages/common/git-show.md

```diff
@@ -22,3 +22,7 @@
 - Show a commit's hash and message in a single line, suppressing the diff output:
 
 `git show --oneline -s {{commit}}`
+
+- Show the contents of a file as it was at a given branch, tag or commit:
+
+`git show {{commit}}:{{path/to/file}}`
```

Comment:
Should it be {{branch|tag|commit}} then ?

Reply:
I pondered the same. Technically any commit-ish [revision specification](https://git-scm.com/docs/gitrevisions) works here, not just branches, tags and commit hashes. I had thought about using `{{reference}}` as a generic term, but maybe `{{revision}}` is clearer. WDYT?

```
- Show the contents of a file as it was at a given revision (e.g. branch, tag or commit):

`git show {{revision}}:{{path/to/file}}`
```

Reply:
Sounds good.

Reply:
What about `{{commit_ish}}`? I think the official git docs use this language.

Reply:
They use lots of different terms, including "revision" (the page I linked to is called "gitrevisions" 😄). So I'd rather go with that since it's more intuitive IMO.

Reply:
Sure, that works for me.

Reply:
Updated. :+1: